elasticdl/python/common/data_reader.py

```diff
@@ -1,9 +1,12 @@
+import math
 import os
 from abc import ABC, abstractmethod
 from contextlib import closing
 
 import recordio
 
+from elasticdl.python.common.odps_io import ODPSReader
+
```

Comment:
should we create a module named `io` or  `reader` instead of putting them into `common` module?

Reply:
Good suggestion. I'll rename them later together since this will cause conflicts with my other work-in-progress branch.